### PR TITLE
[test] Add PrgEnv-nvidia to halo_cell_exchange check

### PR DESCRIPTION
**Pull request by ajocksch**
halo_cell_exchange check PrgEnv-nvidia
https://jira.cscs.ch/browse/UES-1641

### CODE REVIEW

**Review by jgphpc — APPROVED** [review]

**Comment by jjotero** [comment]
@jgphpc There are heaps of failures for this test. At least, the performance references should be adjusted to a less optimistic value.

**Review by jgphpc — CHANGES_REQUESTED** [review]
I get the same failure (in performance) with almost all PrgEnv.
Can we relax the performance limits or is there an issue ?

**Comment by jjotero** [comment]
@jgphpc I don't think there is any performance issue. I just think that the tolerances are way too strict for this case. I also vote for relaxing them.

File: cscs-checks/mch/collectives_halo.py [hunk 1]
```diff
@@ -84,6 +84,7 @@ def __init__(self, bench_reference):
         }
 
         self.maintainers = ['AJ', 'MKr']
+        self.strict_check = False
```
Comment (jjotero) [inline cscs-checks/mch/collectives_halo.py:87]: I think you've added the `strict_check` into the wrong test. This should go into the `halo_cell_exchange.py` instead.

**Review by jjotero — CHANGES_REQUESTED** [review]

**Review by jgphpc — APPROVED** [review]

**Review by jjotero — APPROVED** [review]

**Pull request merged**